### PR TITLE
MAISTRA-2558 regenerate CRDs as v1 instead of v1beta1

### PR DESCRIPTION
**Pull request by rcernich**
Signed-off-by: rcernich <rcernich@redhat.com>

### CODE REVIEW

File: manifests/maistra.io_servicemeshcontrolplanes.yaml [hunk 1]
```diff
@@ -2730,6 +2730,7 @@ spec:
                               format: int32
                               type: integer
                             protocol:
+                              default: TCP
```
Comment (dgn) [inline manifests/maistra.io_servicemeshcontrolplanes.yaml:2733]: I think we previously explicitly removed this from the CRDs due to some k8s bug. Is that resolved with v1?

Comment (rcernich) [reply manifests/maistra.io_servicemeshcontrolplanes.yaml:2733]: Good catch. I don't remember.  I'll need to amend the changes for maistra/istio-operator, as I did not update the operator CRDs to v1.

Comment (jwendell) [reply manifests/maistra.io_servicemeshcontrolplanes.yaml:2733]: yeah, it's fixed in v1.

File: Makefile [hunk 2]
```diff
@@ -60,8 +60,8 @@ security_v1_pb_gos := $(security_v1_protos:.proto=.pb.go)
 docs_path := docs/crd
 
 generate-crd:
-	$(CONTROLLER_GEN) crd:preserveUnknownFields=false,crdVersions=v1beta1 paths=$(path_apis) output:dir=./manifests/
-	sed -i -e '/---/d' ./manifests/maistra.io_*.yaml
+	$(CONTROLLER_GEN) crd:preserveUnknownFields=false,crdVersions=v1 paths=$(path_apis) output:dir=./manifests/
+	sed -i -e '/---/d' ./manifests/maistra.io_*.yaml ./manifests/federation.maistra.io_*.yaml
```
Comment (jwendell) [inline Makefile:64]: nit: `*.yaml` should do the job

**Review by jwendell — APPROVED** [review]

**Comment by rcernich** [comment]
@dgn, are you okay with this?

**Review by dgn — APPROVED** [review]

**Comment by dgn** [comment]
> @dgn, are you okay with this?

absolutely! sorry for the delay

**Pull request merged**